### PR TITLE
Localization: Fix translation deployer

### LINKED ISSUE #529: Translations deployer is not working correctly
We had to change translations structure according to mobile team and that broke our deployer (it's not possible to use it now).

### PR DESCRIPTION
**Pull request by tbergq**
closes #529

### CODE REVIEW

File: packages/localization/src/DefaultVocabulary.js [hunk 1]
```diff
@@ -52,6 +50,31 @@ const Translations = {
...
 export default (Translations: TranslationKeysObject);
+
+export type TranslationPackages =
+  | 'HotelsVocabulary'
+  | 'ManageMyBookingVocabulary'
+  | 'SharedVocabulary';
+
+export const getVocabularies = (vocabularies: TranslationPackages[]) => {
+  const translationPackages = {
+    HotelsVocabulary: { ...HotelsVocabulary },
```
Comment (mrtnzlml) [inline packages/localization/src/DefaultVocabulary.js]: Why is this necessary? Why not just `HotelsVocabulary`? I am not sure I understand it.

**Comment by tbergq** [comment]
@mrtnzlml 🚀

**Comment by tbergq** [comment]
@mrtnzlml 🚀

File: app/hotels/src/singleHotel/hotelInformation/description/Description.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ export class Description extends React.Component<Props> {
           truncatedText={
             <Translation id="single_hotel.description.show_more" />
           }
-          revealedText={<Translation id="single_hotel.description.show_more" />}
+          revealedText={<Translation id="single_hotel.description.show_less" />}
```
Comment (mrtnzlml) [inline app/hotels/src/singleHotel/hotelInformation/description/Description.js]: Hm? Nobody ever noticed this but wrong shade of the smallest grey line are repeatately reported? 😄

**Pull request merged**